health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/ProgressBar.js

```
@@ -0,0 +1,27 @@
+    backgroundColor: bgcolor,
+    width: `${progress}%`,
+  };
+
+  const textStyle = {
+    color: bgcolor,
+  };
+
+  return (
+    <div className="digit-parentDiv-progress">
+      <div className="digit-containerDiv-progress">
+        <div className={"digit-progress-fill"} style={fillerStyle}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)